### PR TITLE
Ignore def for vim9script

### PR DESCRIPTION
**Pull request by ryu-ichiroh**

### CODE REVIEW

**Comment by mattn** [comment]
Please skip comnent tokens.

**Comment by ryuichiroh** [comment]
Which does it mean?

Outside of `def`?
```
" F2 returns ...
def F2()
enddef
```

Or inside?
```
def F2()
# endef
enddef
```

**Comment by ryuichiroh** [comment]
Definitely you said the latter.

**Comment by ryuichiroh** [comment]
@mattn 
I'm finished. Please review the PR again.

https://github.com/vim-jp/vim-vimlparser/blob/4b2f1e250a8db39884269968a98fb153190296f4/test/test_vim9script.vim#L1-L13

**Comment by mattn** [comment]
LGTM, Thank you.

File: autoload/vimlparser.vim [hunk 1]
```diff
@@ -1121,6 +1121,18 @@ function! s:VimLParser.parse_cmd_common() abort
         break
       endif
     endwhile
+  elseif self.ea.cmd.name ==# 'def'
+    " TODO: support vim9script
+    while !self.reader.eof()
+      if self.reader.peek() ==# '#'
+        call self.reader.readline()
+      endif
```
Comment (mattn) [inline autoload/vimlparser.vim:1129]: This should be "else" ? After called readline, seek pos might be already EOF before enddef.

**Pull request closed without merging**